src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.CacheEntry.cs

```diff
@@ -87,7 +87,7 @@ static CompletionItem GetAppropriateAttributeItem(CompletionItem attributeItem,
                     if (attributeItem.DisplayText.TryGetWithoutAttributeSuffix(isCaseSensitive: isCaseSensitive, out var attributeNameWithoutSuffix))
                     {
                         // We don't want to cache this item.
-                        return ImportCompletionItem.CreateAttributeItemWithoutSuffix(attributeItem, attributeNameWithoutSuffix);
+                        return ImportCompletionItem.CreateAttributeItemWithoutSuffix(attributeItem, attributeNameWithoutSuffix, CompletionItemFlags.Expanded);
```

Comment:
why is this guy 'expanded'?  what does that mean?

Reply:
Any item created by this service is "expanded", which means this is from unimported namespace.

Reply:
Rather than passing it in, should this just be hard-coded in ImportCompletionItem.CreateAttributeItemWithoutSuffix?

Reply:
I'd prefer to keep it this way, so it's clear in the cache implementation that those items are not cached. Alternatively, I'm fine moving the `CreateAttributeItemWithoutSuffix` helper to this file.